app/Controllers/CtrlCertificate.php

```diff
@@ -2,13 +2,27 @@
 
 namespace App\Controllers;
 
+use App\Classes\FileValidationConfig;
+use App\Classes\FileValidationConfigBuilder;
 use App\Exceptions\InvalidInputException;
 use App\Models\CertificateModel;
+use App\Utils\FileManager;
 use App\Validation\CertificateValidation;
 use Throwable;
 
-class CtrlCertificate extends BaseController
+class CtrlCertificate extends CtrlApiFiles
 {
+    protected FileValidationConfig $fileConfig;
+
+    public function __construct()
+    {
+        $fileConfigBuilder = new FileValidationConfigBuilder('/admin/certificados');
+        $fileConfigBuilder->builder('certificatePreviewId')->minFiles(1)->maxFiles(1)->maxSize(2, 'MB')->isImage()->maxDims(1000, 1500)->build();
```

Comment:
Why `certificatePreviewId` should be only `certificatePreview`?
And i think the max dims are to much, look a real example
```suggestion
        $fileConfigBuilder->builder('certificatePreview')->minFiles(1)->maxFiles(1)->maxSize(2, 'MB')->isImage()->maxDims(700, 700)->build();
```

![image](https://github.com/loopcrack-org/Kinub/assets/24960025/9862444b-c252-4398-acc4-43f0fce34100)